src/Plugins/ComplexCore/src/ComplexCore/Filters/RawBinaryReaderFilter.hpp

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "ComplexCore/ComplexCore_export.hpp"
+
+#include "complex/Filter/FilterTraits.hpp"
+#include "complex/Filter/IFilter.hpp"
+
+namespace complex
+{
+/**
+ * @class RawBinaryReaderFilter
```

Comment:
Fill some basics about what this filter does for the doxygen. No need to bring in the entire MD file but a statement regarding what this filter does is needed.